shogun-lib/src/main/java/de/terrestris/shoguncore/model/jsonb/ApplicationClientConfig.java

```diff
@@ -5,9 +5,6 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-@Data
-@AllArgsConstructor
-@NoArgsConstructor
-public class ApplicationClientConfig implements Serializable {
-    private String logoPath;
+public interface ApplicationClientConfig extends Serializable {
```

Comment:
Can we also introduce the types for all other JSONB fields on the `Application`?